pkg/session/bootstrap.go

```diff
@@ -3575,7 +3575,7 @@ func rebuildAllPartitionValueMapAndSorted(s *session) {
 			if pi == nil || pi.Type != model.PartitionTypeList {
 				continue
 			}
-			tbl, ok := is.TableByID(s.currentCtx, t.ID)
+			tbl, ok := is.TableByID(context.Background(), t.ID)
```

Comment:
please pass `ctx` inside bootstrapSessionImpl to here

Reply:
Done